compiler/backend/src/org/jetbrains/kotlin/codegen/ErasedInlineClassBodyCodegen.kt

```diff
@@ -109,11 +111,16 @@ class ErasedInlineClassBodyCodegen(
 
 
                 override fun doGenerateBody(codegen: ExpressionCodegen, signature: JvmMethodSignature) {
-                    val iv = codegen.v
-                    iv.aconst(null)
-                    iv.athrow()
+                    val asmType = signature.valueParameters[0].asmType
+                    val secondParameterIndex = asmType.getSize()
```

Comment:
I think you could use `codegen.frameMap` to get JVM parameter index from a parameter descriptor directly. But it probably wouldn't be easier, so I don't insist

Reply:
Oh, thank you! I was actually looking for how to map from parameters to indices and only put in the constants since I didn't find this function... it's fixed now.